internal/spentcalories/spentCalories.go

```diff
@@ -12,7 +16,20 @@ const (
 )
 
 func parseTraining(data string) (int, string, time.Duration, error) {
-	// ваш код ниже
+	parts := strings.Split(data, ",") //Spliting the string into a slice of strings.
+	if len(parts) != 3 {              //Checking the slice length is 3
+		return 0, "", 0, errors.New("invalid format: eptected 3 components")
+	}
+	steps, err := strconv.Atoi(parts[0]) //converting steps into int
+	if err != nil {
+		return 0, "", 0, fmt.Errorf("error of step parsing") //error
```

Comment:
Нужно обернуть ошибку, а то мы просто ее потеряли
```suggestion
		return 0, "", 0, fmt.Errorf("error of step parsing: %w", err) //error
```